### PR TITLE
network information CLI command

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

#819

# Overview

- [x] I have added a `network information` command to the CLI
- [x] I have added a basic CLI integration test
- [x] I have added an API integration scenario for methods not allowed

# Comments

Feel free to add some more tests.

### CODE REVIEW

File: lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs [hunk 1]
```diff
@@ -85,12 +85,13 @@ PrivateKey                             sql=private_key
...
-    txMetaDirection  W.Direction  sql=direction
-    txMetaSlotId     W.SlotId     sql=slot
-    txMetaAmount     Natural      sql=amount
+    txMetaTxId         TxId         sql=tx_id
+    txMetaWalletId     W.WalletId   sql=wallet_id
+    txMetaStatus       W.TxStatus   sql=status
+    txMetaDirection    W.Direction  sql=direction
+    txMetaSlot         W.SlotId     sql=slot
+    txMetaBlockHeight  Word32       sql=block_height
+    txMetaAmount       Natural      sql=amount
```
Comment (KtorZ) [inline lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs:94]: renamed `txMetaSlotId` -> `txMetaSlot` (to be more consistent with other tables) and added `txMetaBlockHeight`

File: lib/core-integration/src/Test/Integration/Scenario/API/Network.hs [hunk 2]
```diff
@@ -32,3 +40,11 @@ spec = do
         eventually $ do
             r <- request @ApiNetworkInformation ctx endpoint Default Empty
             verify r [ expectFieldEqual syncProgress Ready ]
+
+    describe "NETWORK - v2/network/information - Methods Not Allowed" $ do
+        let matrix = ["POST", "CONNECT", "TRACE", "OPTIONS"]
```
Comment (rvl) [inline lib/core-integration/src/Test/Integration/Scenario/API/Network.hs:57]: This is really just testing servant, not our application, isn't it?

**Review by rvl — APPROVED** [review]
It looks fine to me.

File: lib/core-integration/src/Test/Integration/Scenario/API/Network.hs (hunk 2 above)
Comment (KtorZ) [reply lib/core-integration/src/Test/Integration/Scenario/API/Network.hs:57]: To some extend yes, it verifies that our setup matches our expectations. A blackbox test if you prefer.

**Review by paweljakubas — APPROVED** [review]
LGTM!

File: lib/core-integration/src/Test/Integration/Scenario/API/Network.hs (hunk 2 above)
Comment (paweljakubas) [reply lib/core-integration/src/Test/Integration/Scenario/API/Network.hs:57]: maybe this blog post is of some interest to us : https://blog.ploeh.dk/2019/09/23/unit-testing-wai-applications/

**Review by piotr-iohk — APPROVED** [review]

**Pull request merged**